ArticleTemplates/assets/js/bootstraps/campaign.js

```diff
@@ -0,0 +1,68 @@
+define([], function () {
+    'use strict';
+
+    const rootPath = 'https://code.api.nextgen.guardianapps.co.uk/formstack-campaign/submit'
```

Comment:
Unfortunately `const` only has partial support on older versions of Android that we're still supporting, so we'll need to convert to `var`. The same goes for `let` which has no support on older versions.

Reply:
The JS validation (`yarn validate`) will complain about a missing semicolon here as well.